src/components/Product/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+
+import { formatNumber } from "../../helper";
+import btnCart from "../../assets/Icon/btn_cart.svg";
+
+import "./style.scss";
+import { Row, Col } from "antd";
+const Product = ({ item, showOption }) => {
```

Comment:
```suggestion
const Product = ({ item, showOption = false }) => {
```
các giá trị boolean thì nhớ thêm default value vào nhé